### PR TITLE
Unix timestamp and SHA256 Generator

### PR DESCRIPTION
**Pull request by cvadstrup**
- Codeunit for creating unix timestamp 
- Codeunit for SHA256 cryptography support

### CODE REVIEW

File: SHA256.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  {
+    Date=06-09-17;
+    Time=06:27:38;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+            MESSAGE(CalcHMAC('SHA256 HASH Generator','iso-8859-1'));
```
Comment (BardurKnudsen) [inline SHA256.txt]: Add a test codeunit instead.

File: SHA256.txt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  {
+
+    PROCEDURE CalcHMAC@6150652(HMACString@6150613 : Text[1024];CharSet@6150615 : Text[30]) : Text[100];
+    VAR
+      Encoding@6150616 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Text.Encoding";
+      MerchantKey@6150617 : Text[50];
+      HashValue@6150614 : Text[100];
+    BEGIN
+      Encoding := Encoding.GetEncoding(CharSet);
+      MerchantKey := '123456789';
```
Comment (BardurKnudsen) [inline SHA256.txt]: Should this be a parameter? Seems like a test value?

File: Unix Time.txt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  {
+    OnRun=BEGIN
+            MESSAGE(FORMAT(GetUnixTime(CURRENTDATETIME)));
+          END;
+
+  }
+  CODE
+  {
+
+    PROCEDURE GetUnixTime@6150645(FromDateTime@6150614 : DateTime) UnixTimeStamp : Integer;
```
Comment (BardurKnudsen) [inline Unix Time.txt]: Did you consider adding the function to codeunit 10 instead  - and write it using a similar way as the function EvaluateUnixTimestamp(Timestamp : BigInteger) : DateTime ?

**Comment by BardurKnudsen** [comment]
We believe that we covered the SHA256 hashing by the functions added to codeunit 1266, and we believe that the unix time stamp can be implemented as in codeunit 10.

**Comment by JesperSchulz** [comment]
The new library functions will ship with our November update. Thank you again for your contribution!

**Pull request closed without merging**